deps/rax/rax.h

```diff
@@ -32,6 +32,7 @@
 #define RAX_H
 
 #include <stdint.h>
+#include <stdlib.h>
```

Comment:
Which function required stdlib?

Reply:
Strangely, the definition of `size_t` did.